.github/workflows/ms.analysisservices.servers.yml

```diff
@@ -123,6 +138,7 @@ jobs:
     if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master' || github.event.inputs.prerelease == 'true'
```

Comment:
We shouldn't allow publishing unless both static and deployment validation run. Is that the case?

Reply:
It is. The job depends in the others without any special conditions that would change that

Reply:
Also retested it again running with the pre-release flag alongside only pester vs only deployment for both GitHub & ADO